scipy/optimize/tests/test_linprog.py

```diff
@@ -754,7 +772,7 @@ def test_zero_column_2(self):
                       method=self.method, options=self.options)
         _assert_unbounded(res)
         # Unboundedness detected in presolve
-        if self.options.get('presolve', True):
+        if self.options.get('presolve', True) and "highs" not in self.method:
```

Comment:
It appears that HiGHS presolve may be correct here and SciPy is wrong.  @mdhaber can you take a look at https://github.com/ERGO-Code/HiGHS/issues/648 and say if we think scipy presolve is deducing unboundedness too soon?

Reply:
There message the user will get in this case is:
> If feasible, the problem is (trivially) unbounded due to a zero column in the constraint matrices. If you wish to check whether the problem is infeasible, turn presolve off.

So I guess we are doing what a lot of solvers do, reporting "infeasible or unbounded", in a sense.

Reply:
Okay, so it looks correct to exclude highs' methods this check.  Does this need a note or is it good as is?